composer.json

```diff
@@ -34,8 +34,8 @@
         "laravel": {
```

Comment:
we should also add  a `UserServiceProvider` that uses `dynamicMethods` so that we can get a user's events and calendars. The method/relationship for events should be `calendarEvents` so that it's more specific than some event model we could have in an app.